rwgame/RWGame.cpp

```diff
@@ -414,26 +415,28 @@ int RWGame::run() {
 
         auto now = chrono::steady_clock::now();
         auto deltaTime = chrono::duration<float>(now - lastFrame).count();
+        deltaTime *= world->state->basic.timeScale;
         lastFrame = now;
 
-        if(!world->isPaused()) {
+        if (!world->isPaused()) {
             accumulatedTime += deltaTime;
-            world->dynamicsWorld->stepSimulation(deltaTime * timescale, kMaxPhysicsSubSteps, kPhysicsTimeStep);
+            world->dynamicsWorld->stepSimulation(deltaTime, kMaxPhysicsSubSteps,
+                                                 deltaTime);
         }
 
         RW_PROFILE_BEGIN("Update");
-        while (accumulatedTime >= GAME_TIMESTEP && !world->isPaused()) {
+        while (accumulatedTime >= deltaTime && !world->isPaused()) {
```

Comment:
Still do you think there's need with tickrate 60Hz?

Reply:
I'll open another PR for this.